apps/web/src/views/BuyCrypto/components/AccordianDropdown/AccordianItem.tsx

```diff
@@ -155,14 +158,14 @@ function AccordionItem({
   return (
     <Flex flexDirection="column">
       <CryptoCard
-        padding="12px 12px"
+        padding="16px 16px"
         style={{ height }}
         onClick={!isActive() ? toogleVisiblity : () => null}
```

Comment:
Why don't we use precomputed value instead of function call here? Also keep in mind that on each render, the `() => null` will always be a new function instance which will cause the `CryptoCard` component to rerender. The best practice would be defining a `onClick` function with `useCallback`